cns/restserver/ipam.go

```diff
@@ -427,6 +427,7 @@ func (service *HTTPRestService) removeEndpointState(podInfo cns.PodInfo) error {
 	service.Lock()
 	defer service.Unlock()
 	logger.Printf("[removeEndpointState] Removing endpoint state for infra container %s", podInfo.InfraContainerID())
+	logger.Printf("[removeEndpointState] Endpoint state dump: [%+v]", service.EndpointState)
```

Comment:
let's not log the whole internal state in the hot path

Reply:
addressed in the commit